config/config.go

```diff
@@ -42,11 +42,16 @@ func NewConfig(dir string) *Config {
 	return cfg
 }
 
-// TrackingDBDir returns the path of the folder where completed uploads are tracked.
-func (c *Config) TrackingDBDir() string {
+// CompletedUploadsTrackingDBDir returns the path of the folder where completed uploads are tracked.
+func (c *Config) CompletedUploadsTrackingDBDir() string {
 	return path.Join(c.ConfigPath, "uploads.db")
 }
 
+// UploadURLsTrackingDBDir returns the path of the folder where upload URLs are tracked.
+func (c *Config) UploadURLsTrackingDBDir() string {
+	return path.Join(c.ConfigPath, "uploadsurls.db")
```

Comment:
I don't like to have `uploads.db` and `uploadsurls.db`. They are very similar, but changing the first one would be a breaking change and I can't figure out a better name for the second one... maybe `resumable_uploads.db`

Reply:
Agreed, done.